server/src/routes/schedule.js

```diff
@@ -95,30 +95,24 @@ router.get('/candidate/:uuid', (req, res) => {
 });
 
 // add a new user in either the candidates table or interview table based on the selected type
-router.post('/newuser', (req, res) => {
-  const user = req.body;
-  const type = user.role;
-  // status Active as default when adding
-  const status = 'A';
-  const uuid = uuidv1();
-  let sql = '';
-  switch (type) {
-    case 'candidate':
-      sql = 'INSERT INTO Candidate(firstName, lastName, email, phone, status, uuid) VALUES (?, ?, ?, ?, ?, ?)';
-      break;
-    case 'interviewer':
-      sql = 'INSERT INTO Interviewer(firstName, lastName, email, phone, status) VALUES (?, ?, ?, ?, ?)';
-      break;
-    default: return;
+
+router.post('/newcandidate', notAuthMiddleware, (req, res) => {
```

Comment:
As we're changing the endpoint, it would be nice to change it to follow some of the best practices of REST API design. This is definitely a low priority: https://docs.microsoft.com/en-us/azure/architecture/best-practices/api-design#define-operations-in-terms-of-http-methods